src/options.h

```diff
@@ -2524,6 +2524,18 @@ align_func_params_gap;
 extern Option<bool>
 align_same_func_call_params;
 
+// The span for aligning function-call parameters for single line functions.
+//
+// 0 = Don't align (default)
+extern BoundedOption<unsigned, 0, 5000>
+align_same_func_call_params_span;
+
+// The threshold for aligning function-call parameters for single line
+// functions.
+// 0 = No limit (default)
```

Comment:
Not: should probably be a "blank" line here?

```
// The threshold for aligning function-call parameters for single line
// functions.
//
// 0 = No limit (default)
```

Reply:
Actually, another nit: we aren't quite consistent with whether or not there should be a period at the end of e.g. `0 = No limit (default)`, but we use one more often than not, so please add that also. (I'm going to make another PR shortly to clean up the inconsistency.)